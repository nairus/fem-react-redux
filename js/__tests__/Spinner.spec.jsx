import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../Spinner';

test('Spinner render correctly', () => {
  const component = shallow(<Spinner />);
  expect(component).toMatchSnapshot();
});
