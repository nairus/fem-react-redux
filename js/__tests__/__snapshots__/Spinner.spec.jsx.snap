// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner render correctly 1`] = `
<styled.img
  alt="loading indicator"
  src="/public/img/loading.png"
/>
`;
